### DISCUSSION TITLE
Support of M1 Chip on Apple devices

### DISCUSSION THREAD

**Discussion by yermak** [yermak/AudioBookConverter#586] [MacOS]
Current only intel architecture is supported.
Is M1 support required.

Please vote via comments.

**Comment by tamaracks** [comment]
I need M1 support.

**Comment by micweeks** [comment]
I need it

**Comment by letternews** [comment]
me too

**Comment by TIMwilltell** [comment]
Yes please

**Comment by andy-g** [comment]
Yes please

**Comment by yermak** [comment]
Duplicates #580

**Comment by yermak** [comment]
Beta for Mac OS X is available
https://github.com/yermak/AudioBookConverter/issues/580#issuecomment-2268738667
